client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.js

```diff
@@ -30,6 +32,7 @@ const DataFilesProjectMembers = ({
 
   const userSearch = e => {
     if (!e.target.value || e.target.value.trim().length < 1) return;
+    setInputUser(e.target.value);
```

Comment:
This should probably be the first line in the method- right now it's impossible to delete the first letter of a query.